test/robot/report/client.go

```diff
@@ -95,8 +123,20 @@ func doReport(ctx context.Context, action string, in *Input, store *stash.Client
 	if err := store.GetFile(ctx, in.Gapis, gapis); err != nil {
 		return nil, err
 	}
+
+	if in.GetLayout() != nil {
```

Comment:
Oops, actually I should get rid of the TODO in replay/client.go. The `gapidAPK` is necessary for Vulkan replay as gapis will check if the apk is installed on the device, and if such an apk is not found , gapis will just give up (it will not just use whatever apk already installed on device).

The TODO was initially added for the extra apk passed in specifically for querying device info. That is not required and dropped in 0ccbadc (Err, just rebased...)

Reply:
TODO removed in 952fd2a